test/functional/feature_llmq_hpmn.py

```diff
@@ -283,6 +283,10 @@ def test_getmnlistdiff_base(self, baseBlockHash, blockHash):
         assert_equal(set([QuorumId(e["llmqType"], int(e["quorumHash"], 16)) for e in d2["deletedQuorums"]]), set(d.deletedQuorums))
         assert_equal(set([QuorumId(e["llmqType"], int(e["quorumHash"], 16)) for e in d2["newQuorums"]]), set([QuorumId(e.llmqType, e.quorumHash) for e in d.newQuorums]))
 
+        blockchain_info = self.nodes[0].getblockchaininfo()
+        if blockchain_info["softforks"]["v19"]["active"]:
+            assert_equal(blockchain_info["softforks"]["v19"]["height"], d.v19activationHeight)
+
```

Comment:
why in `feature_llmq_hpmn.py`? should also check that p2p matches `protx diff` rpc output. pls see e90da3ad8a